instrumentation/opentelemetry-instrumentation-asgi/src/opentelemetry/instrumentation/asgi/__init__.py

```diff
@@ -660,6 +670,19 @@ async def otel_send(message):
                         setter=asgi_setter,
                     )
 
+                if message.get("headers"):
```

Comment:
I'm wondering if we can just use the get function of ASGIGetter here
It looks like the same logic.
asgi_getter.get("content-length")

Reply:
You're right. I've replaced it with `asgi_getter`.
Thanks.